resource/helpers/runnable-helper.py

```diff
@@ -1,37 +1,74 @@
 import dataiku
 from dku_deeplearning_image.applications import APPLICATIONS
+import api_designer_utils.utils as utils
 
 api_client = dataiku.api_client()
 
+
+def get_code_options():
+    choices = [{
+        "label": "Use an existing...",
```

Comment:
The label should be `Select an environment` to conform with the wording already used inside dss

Reply:
Fixed in 9f4457b